src/vm/moar/QAST/QASTOperationsMAST.nqp

```diff
@@ -2729,7 +2729,6 @@ my constant SIMPLE_OP_MAPPINGS := nqp::list_s(
     'resume', 'resume',
     'lastexpayload', 'lastexpayload',
     'throwextype', 'throwcatdyn',
-    'getarg_i', 'getarg_i',
```

Comment:
Is this supposed to be part of this PR?

Reply:
Ah, yes, it's part of new-disp-nativecall. The PR's name and description are no longer really fitting. They were auto-populated by Github when the branch only contained the first commit.